galaxy/values.yaml

```diff
@@ -4,7 +4,7 @@
 
 image:
   repository: galaxy/galaxy-k8s
-  tag: 20.05
+  tag: 20.05.1
```

Comment:
Is there are point to keep the non-functional 20.05 (previous image)?

Reply:
In this case, it does make sense yes. Have updated the image.